x-pack/plugins/ml/public/maps/util.ts

```diff
@@ -104,47 +100,41 @@ export async function getResultsForJobId(
         typical = getCoordinates(typicalCoordStr);
         typicalDisplay = getCoordinates(typicalCoordStr, true);
       }
-      return {
-        fieldName: _source.field_name,
-        functionDescription: _source.function_description,
-        timestamp: formatHumanReadableDateTimeSeconds(_source.timestamp),
-        typical,
-        typicalDisplay,
-        actual,
-        actualDisplay,
-        record_score: Math.floor(_source.record_score),
-      };
-    });
-  }
 
-  const features: Feature[] = hits.map((result) => {
-    let geometry: Geometry;
-    if (locationType === 'typical' || locationType === 'actual') {
-      geometry = {
-        type: 'Point',
-        coordinates: locationType === 'typical' ? result.typical : result.actual,
-      };
-    } else {
-      geometry = {
-        type: 'LineString',
-        coordinates: [result.typical, result.actual],
+      let geometry: Geometry;
+      if (locationType === 'typical' || locationType === 'actual') {
```

Comment:
nit: I think we can use the new ML_ANOMALY_LAYERS here

Reply:
Yep - missed it! thanks - 1ae5ddce243e4b808f740b5f71349a0333300fce